assets/other-scripts/emails/index.js

```diff
@@ -52,6 +52,18 @@ const ReaderRevenueEmailSidebar = compose( [
 				isDismissible: false,
 			} );
 		}
+		createNotice(
+			'info',
+			sprintf(
+				/* translators: 1: "From" email address 2: "From" email name */
+				__( 'This email will appear as sent from "%1$s" email address, by "%2$s".', 'newspack' ),
+				config.from_email || newspack_emails.from_email,
```

Comment:
That's intentional. Here we want the `from_email` from `config` (for _this_ email configuration) or fall back to `newspack_emails` (global configuration).

Reply:
Whoops, I think I messed up my suggestions—I meant to flip the `from_name` and `from_email` substitutions only because my [copy suggestion](https://github.com/Automattic/newspack-plugin/pull/1937#discussion_r958947240) changed their order. Sorry for the confusion.

Reply:
Ah, got it. Fixed in https://github.com/Automattic/newspack-plugin/pull/1961